ddclient

```diff
@@ -2184,6 +2157,10 @@ sub get_ip {
 
     if ($use eq 'ip') {
         $ip  = opt('ip', $h);
+        if (!is_ipv4($ip) && !is_ipv6($ip)) {
+            warning("'%s' is not a valid IPv4 or IPv6 address, ignoring value.",$ip);
```

Comment:
"Ignoring value" might be misleading depending on what the caller does with `undef`.

```suggestion
            warning("'%s' is not a valid IPv4 or IPv6 address", $ip);
```